examples/Phonon_Caustics/README.md

```diff
@@ -0,0 +1,116 @@
+
+In our simulation, we follow the approach of creating a big detector, because it is significantly more computationally efficient.. 
+
+
+ The phonon simulation is simple in terms of geometry. The basic elements of the simulation are 
+
+* A sapphire substrate of 4mm x 4mm x 4mm.
+* A big bolometer sensor of 4mm x 4mm x 0.0001 mm (Sensitive part where the phonons are absorbed).
+* The new config.txt file with all the parameters for the sapphire substrate.
+* The Caustic.mac file to generate 40e+6 phonos with isotropic initial random momentum.
+* The Caustic_plot.root program to plot the caustic pattern for Transverse Slow Phonons, Transverse Fast Phonons, or both together.
+# Substrate Dimension to Match the Experimental Phonon Caustics
+The phonon caustics depends on the angular dimension and the size of the subtrate in exeprimental. <br> 
+In order to match the experimental angles with the simualtion we need to initialize the initial phonon at difference distance <br> from the top part of the bolometers.
+The angle is differents for each material. You need to use the following dimensions and the initial position of the phonons to reproduce the experimental phonon caustisc.<br> 
+* Sapphire 4mm x 4mm x 4mm ```console /gps/pos/centre 0.0 0.0 0.08 cm ``` , &theta; = 58.0
+* GaAs 4mm x 4mm x 4mm ```console /gps/pos/centre 0.0 0.0 0.08 cm ``` , &theta; = 59.04
+* LiF 4mm x 4mm x 4mm ```console /gps/pos/centre 0.0 0.0 -0.03 cm ``` , &theta; = 40.0
+* CaF2 4mm x 4mm x 8mm ```console /gps/pos/centre 0.0 0.0 -0.06 cm ``` , &theta; = 23.0
+* CaWO4 9mm x 9mm x 6mm ```console /gps/pos/centre 0.0 0.0 0.0 cm ``` , &theta; = 56.3
+
+# Miller Orientations
+The phonon caustic depends on the orientation of the crystal with the sensor. You can obtain different phonon caustics images by changing the crystal orientation. <br>
+In the case of Sapphire 4 Miller indices are needed. The program 4_Miller_to_3_Miller.py converts 4 Miller indices to 3 Miller.<br>
+For more information check this link   [4 Miller indices ](https://apps.dtic.mil/sti/trecms/pdf/AD1115835.pdf)<br>
+Note.-  The program 4_Miller_to_3_Miller.py is designed to use only for sapphire and match the [experimental results](https://journals.aps.org/prb/abstract/10.1103/PhysRevB.29.2190) .
+
+# How to Obtain the Phonon Caustics Patterns
+We follow the standard procedure established on Geant4 to compile and execute a program. 
+```console
+User@fun:~$ cmake ..
+User@fun:~$ make install
+[100%] Built target g4cmpPhononCaustics
+```
+If you do not have errors
+```console
+./g4cmpaSapphire Caustic.mac
+```
+You will have a txt file "phonon_hits.txt" with three columns: <br> 
```

Comment:
Please include a comment, either here or somewhere nearby, that the example does not currently support running with multiple threads.  The user should ensure that `/run/numberOfThreads 1` is specified in their macro file.

Reply:
I include the following statement on the Readme file 
This example does not support running with multiple threads. The user should ensure that  /run/numberOfThreads 1  is specified in their macro file.